packages/cli/create-strapi-app/create-strapi-app.js

```diff
@@ -56,7 +56,7 @@ function generateApp(projectName, options) {
 }
 
 async function initProject(projectName, program) {
-  await checkInstallPath(resolve(projectName));
+  if (projectName) await checkInstallPath(resolve(projectName));
```

Comment:
shouldn't you just move the 82 line  at the beginning of the function and checkInstallPath of the directory var ?